Sources/MediaXDMEventGenerator.swift

```diff
@@ -308,17 +308,17 @@ class  MediaXDMEventGenerator {
     private func getReportingIntervalFromTrackerConfig(isAdStart: Bool = false) -> Int {
```

Comment:
Would it work if the return type was `TimeInterval`?  That way you wouldn't need to cast to `Int` when using the result in `processPlayback`.